dspace-api/src/main/java/org/dspace/handle/dao/impl/HandleDAOImpl.java

```diff
@@ -83,7 +83,7 @@ public long countHandlesByPrefix(Context context, String prefix) throws SQLExcep
     @Override
     public int updateHandlesWithNewPrefix(Context context, String newPrefix, String oldPrefix) throws SQLException
     {
-        String hql = "UPDATE handle set handle = concat(:newPrefix,'/',id WHERE handle like :oldPrefix%";
+        String hql = "UPDATE Handle set handle = concat(:newPrefix,'/',handle_id) WHERE handle like concat(:oldPrefix,'%')";
```

Comment:
@abollini @tdonohue you are right! I do not realize of this. I've replaced the "handle_id" by the "id" property and it works fine!